openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -57,10 +58,21 @@ def process(self, instance):
         else:
             members = instance[:]
 
+        loaded_containers = None
+        if set(self.add_for_families).intersection(
+                set(instance.data.get("families")),
+                set(instance.data.get("family").lower())):
```

Comment:
1) `set(instance.data.get("family").lower())` this is incorrect? Family is not a list and thus this would turn into e.g. `set("model".lower())` which turns into `set("model")` which becomes `{"m", "o", "d", "e", "l"}` :)
2) `.lower()` can be removed?

Reply:
Also, are you sure you require the `family` to also be present in `families` which is what the current logic is doing because `a.intersection(b, c)` returns a set with items that is present in all 3 sets.

I'd rather detect whether it's at least present in either as in [my pseudocode mockup](https://github.com/pypeclub/OpenPype/pull/2837#discussion_r824517669)